woocommerce/woocommerce-android
# Hide FAB when bottom navigation bar is animating

Closes: #7986


### Description

Fixes laggy animation when a user navigates to another screen from either orders or products fragments

The solution is quite messy. I could not find anything better =(

### Testing instructions

* Open orders
* Click on the FAB
* Notice that there is no laggy animation

---
* Open products
* Click on a product 
* Notice that there is no laggy animation

### Images/gif

#### Before
https://user-images.githubusercontent.com/4923871/205645174-695fb19b-729d-4e06-a51d-06c27037a50b.mp4

#### After
https://user-images.githubusercontent.com/4923871/205645228-5047db64-b63e-489f-88eb-533915454cdf.mp4
